app/models/Category.py

```diff
@@ -6,12 +6,13 @@ class Category(db.Model):
     A category of a post.
     """
 
+    __searchable__ = ['name']
```

Comment:
Is this for the [Flask-WhooshAlchemy](https://pythonhosted.org/Flask-WhooshAlchemy/) extension? If so, you need to add a requirement in `requirements.txt`.

Reply:
oh yeah, oops, I forgot to remove that >_<